app/controllers/v1/tiles_controller.rb

```diff
@@ -0,0 +1,40 @@
+class V1::TilesController < ApplicationController
+  def create
+    tile = Tile.new(create_params)
```

Comment:
Can anyone create tiles on any project? Or should this be enforcing project_id is in current_user.projects?